tools/crm_resource_ban.c

```diff
@@ -246,8 +246,16 @@ resource_clear_node_in_expr(const char *rsc_id, const char *host, cib_t * cib_co
     int rc = pcmk_rc_ok;
     char *xpath_string = NULL;
 
-    xpath_string = crm_strdup_printf("//rsc_location[@id='cli-prefer-%s'][rule[@id='cli-prefer-rule-%s']/expression[@attribute='#uname' and @value='%s']]",
-                                     rsc_id, rsc_id, host);
+#define xpath_fmt                                                       \
```

Comment:
caps for consistency

Reply:
Whoops, yep. This is because at first I used a `const char *xpath_fmt`, and the compiler complained because of `-Wformat-nonliteral`.